sdk/python/kfp/components/tasks_group.py

```diff
@@ -171,18 +171,23 @@ class ParallelFor(TasksGroup):
 
     Args:
         items: The items to loop over. It can be either a constant Python list or a list output from an upstream task.
-        name: The name of the for loop group.
-        parallelism: The maximum number of concurrent subtasks that can be scheduled for execution.
+        name: Optional; the name of the for loop group.
```

Comment:
Can you remove the use of `Optional`? We've opted to not add any type or optional/required information to the docstrings, since this is fully represented by the class/function signatures.

The codebase isn't fully compliant yet, so there's no way you would know this :)

Reply:
Done